webtau-graphql/src/main/java/org/testingisdocumenting/webtau/graphql/ConfigBasedGraphQLHttpConfiguration.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2021 webtau maintainers
+ * Copyright 2019 TWO SIGMA OPEN SOURCE, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.testingisdocumenting.webtau.graphql;
+
+import static org.testingisdocumenting.webtau.cfg.WebTauConfig.getCfg;
+
+import org.testingisdocumenting.webtau.graphql.config.GraphQLHttpConfiguration;
+import org.testingisdocumenting.webtau.graphql.model.GraphQLRequest;
+
+public class ConfigBasedGraphQLHttpConfiguration implements GraphQLHttpConfiguration {
+    @Override
+    public String requestUrl(String url, final GraphQLRequest graphQLRequest) {
+        String endpoint = getCfg().get(GraphQLConfig.graphQLEndpoint.getKey());
```

Comment:
Why this approach to getting this config value? I think you could just do `GraphQLConfig.graphQLEndpoint.getAsString()` or maybe better to follow the pattern we have with `GraphQLConfig.ignoreIntrospectionFailures()`.

Reply:
not sure why I didn't see that, fixed